scripts/serve.sh

```
@@ -0,0 +1 @@
+docker compose -f docker-compose.yml -f docker-compose.dev-override.yml up --build --watch
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)